### DISCUSSION TITLE
How do I have Home/End move the cursor in the input area?

### DISCUSSION THREAD

**Discussion by lunagloaming** [Blightmud/Blightmud#858] [Q&A] [answered]
Another help request... the home key rather uselessly takes me to the start of the log.  It would be nice if it affected the cursor position in the input area, like most programs.  Ctrl-PgUp (or another meta key) could replace the old Home functionality.

**Comment by LiquidityC** [comment, marked as answer]
Blightmud defaults to terminal standards for cursor navigation. `ctrl-a` and `ctrl-e` for home and end. 

You can change this as you like with bindings. Take a look at `/help bindings`

**Comment by lunagloaming** [comment]
Interesting.  Thank you, that does make its own sort of sense.

**Comment by lunagloaming** [comment]
```
/help bindings
```
This shows LUA functions again, with no help how a user is supposed to trigger them.

**Reply by LiquidityC** [reply]
`/help config_scripts` and `/help scripting` should give you a quick introduction into how to script in blightmud using lua.

**Reply by LiquidityC** [reply]
`/help <word>` will search all help files for occurrences of that word
